frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -382,7 +382,8 @@ function roleSelected(evt: any) {
                     <div v-if="forestClientData" class="row">
                         <div class="col-md-5 px-0">
                             <ForestClientCard
-                                :text="forestClientData?.[0]?.client_name"
+                                :clientId="forestClientData?.[0]?.forest_client_number"
+                                :clientName="forestClientData?.[0]?.client_name"
                                 :status="forestClientData?.[0]?.status"
```

Comment:
Why don't we redefine <ForestClientCard> props to take the "forestClientData" (since it passes all the properties from the object: "client_name", "forest_client_number" and "status")? Unpacking them seems not necessary.

Reply:
Hello! Sure, I'll do that!